### DISCUSSION TITLE
Replicator set types not prefixed with tab

### DISCUSSION THREAD

**Discussion by Web10-Joris** [statamic/cms#9881] [Troubleshooting]
Is it possible to have replicator sets prefixed with the tab they are displayed in? 

Now, when you have different tabs with multiple sets in the replicator fieldtype, the `type` of the set is always just the handle. Is it possible to prefix this with the tab, or add a new variable inside the content of the sets with its tab name/handle? 

Example: I have a tab in my replicator field like `elements`. Inside that is a set called `buttons`. The `type` of this selected set only returns `buttons` but should return `elements.buttons`

**Comment by edalzell** [comment]
That's not possible right now, what is your use case? Why do you need the prefix?

**Reply by Web10-Joris** [reply]
Every set inside the replicator has its own view, prefixing it with the tab structures the views for these sets. Now I'm loading the views using the partial tag & type variable, but would be nice to structure the views even more in subfolders per tab. Is it maybe possible to programmatically retrieve or add this in a separate variable for each set?
